images/hook/entrypoint.sh

```diff
@@ -116,8 +61,15 @@ spec:
                 values:
                 - $NODE_NAME
 EOF
-    kubectl --namespace=monitoring patch deployment influxdb -p "$(cat $TMPFILE)"
-    rm $TMPFILE
+        kubectl --namespace=monitoring patch deployment influxdb -p "$(cat $TMPFILE)"
+        rm $TMPFILE
+    fi
+
+    echo "---> Checking status of updated resources"
+    rig status $RIG_CHANGESET --retry-attempts=120 --retry-period=1s --debug
```

Comment:
Is there a particular reason we check the status here as well as below after all resources have been created?

Reply:
The intention of this status check is to check equality of current and desired state of resources.